### PR TITLE
Adding the SeePassword in the page.tsx file 

### PR DESCRIPTION
**Pull request by Suvendu-UI**
# Pull Request Template

## How
Explain how you implemented the changes. Include any relevant information that would help reviewers understand your approach.

I added seePassword function which changes the type in input from "text" to "password" using onClick

### CODE REVIEW

**Comment by Suvendu-UI** [comment]
Hey @manuelalferez new pr pls review this

File: src/app/signin/auth-ui.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
 "use client";
 
-import { getURL } from "@/lib/utils";
+import { getURL } from "../../lib/utils";
```
Comment (manuelalferez) [inline src/app/signin/auth-ui.tsx:3]: Please, use absolute imports. We are using absolute imports in all the project. 
```tsx
import { getURL } from "@/lib/utils";
```

**Comment by manuelalferez** [comment]
Why is the view password button outside the password input? 

<p align="center">
  image
</p>

It is good practice to have the components and their most direct functionality as close to each other as possible. Example:

https://github.com/user-attachments/assets/bc4a975b-e3c5-40d6-83d6-e94c15e19bad

This page of components are very good, and they already do the work for you:
https://originui.com/inputs

**Comment by manuelalferez** [comment]
@Suvendu-UI I don't think I understand what you are trying to say or ask. please can you rephrase? I really want to understand you

**Comment by Suvendu-UI** [comment]
i can't get the .tsx file for this part so that i can put the button
Screenshot 2024-10-18 211357

**Comment by Suvendu-UI** [comment]
it would be nice if you could tell me the file where this part's code is

**Comment by Suvendu-UI** [comment]
made the changes just the code the place where there is the input of password of email then we are set to go

**Comment by manuelalferez** [comment]
Hi @Suvendu-UI, the show button is outside the component. It should not be there, it should be inside the password input. Can you move it there? 
image

**Comment by Suvendu-UI** [comment]
I know this but I couldn't get the place where the location of that file having the input for password and email is there, can you give the location of the file ?
